crates/shared/src/api.rs

```diff
@@ -218,7 +218,16 @@ pub fn finalize_router(
     let cors = warp::cors()
         .allow_any_origin()
         .allow_methods(vec!["GET", "POST", "DELETE", "OPTIONS", "PUT", "PATCH"])
-        .allow_headers(vec!["Origin", "Content-Type", "X-Auth-Token", "X-AppId"]);
+        .allow_headers(vec![
```

Comment:
Should be reverted before merging. Not sure why but this was needed to make this work with the local setup. When this is running in the actual infra this should not be necessary.